src/components/NostoProduct.tsx

```diff
@@ -28,10 +27,9 @@ import { useNostoApi } from "../utils/hooks"
  */
 export default function NostoProduct(props: {
   product: string
-  tagging?: Product
```

Comment:
I would try to utilize this in the Session API call, e.g.

api.defaultSession().viewProduct(tagging ?? product)